### PR TITLE
disabling console logger by default

### PR DESCRIPTION
**Pull request by partnerinflight**
no need to rip it out entirely, we should just default to off

### CODE REVIEW

**Comment by oldalton** [comment]
Build is failing :/ please fix before merging

**Comment by partnerinflight** [comment]
yeah, not merging on a red build. :)

**Comment by unpluggedk** [comment]
I am with @RPangrle , but am OK with doing it later.

File: MSAL/src/MSALLogger.m [hunk 1]
```diff
@@ -58,7 +58,6 @@ - (id)init
     // and we'll probably not have enough diagnostic information, however verbose
     // will most likely be too noisy for most usage.
     self.level = MSALLogLevelInfo;
-    self.consoleLogging = NO;
```
Comment (oldalton) [inline MSAL/src/MSALLogger.m]: There's still consoleLogging property in the MSALLogger header file

Comment (RPangrle) [inline MSAL/src/MSALLogger.m]: Yes.

**Comment by RPangrle** [comment]
Fixes #158

**Pull request merged**